storage: TestRangeQuiescence is flaky

Seen in https://teamcity.cockroachdb.com/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=23283#_focus=82088.

Also finding this locally on my WIP.

```
I160927 16:13:36.929657 1325 /Users/tschottdorf/go/src/google.golang.org/grpc/clientconn.go:667  grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:64262: operation was canceled"; Reconnecting to {"127.0.0.1:64262" <nil>}
I160927 16:13:36.929908 1325 /Users/tschottdorf/go/src/google.golang.org/grpc/clientconn.go:767  grpc: addrConn.transportMonitor exits due to: grpc: the connection is closing
I160927 16:13:36.930499 13177 util/stop/stopper.go:353  stop has been called, stopping or quiescing all running tasks
I160927 16:13:36.931100 13177 util/stop/stopper.go:353  stop has been called, stopping or quiescing all running tasks
I160927 16:13:36.931675 13177 util/stop/stopper.go:353  stop has been called, stopping or quiescing all running tasks
--- FAIL: TestRangeQuiescence (1.14s)
       	client_raft_test.go:2640: [n1,s1,r1:/M{in-ax}] should be the leader: StateFollower
FAIL


ERROR: exit status 1

232 runs completed, 1 failures, over 53s
FAIL
make: *** [stress] Error 1
```

Nope, this is just flaky on master.

```
W160927 20:48:22.564183 515 storage/raft_transport.go:440  raft transport stream to node 2 failed: rpc error: code = 13 desc = transport is closing
--- FAIL: TestRangeQuiescence (0.43s)
       	client_raft_test.go:2640: [n1,s1,r1:/M{in-ax}] should be the leader: StateFollower
FAIL


ERROR: exit status 1

2030 runs completed, 1 failures, over 14s
FAIL
Makefile:146: recipe for target 'stress' failed
make: *** [stress] Error 1
✘-2 ~/src/github.com/cockroachdb/cockroach [master|⚑ 1]
```

--- a/storage/client_raft_test.go
+++ b/storage/client_raft_test.go
@@ -2635,7 +2635,7 @@ func TestRangeQuiescence(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 
 	sc := storage.TestStoreConfig()
-	sc.RaftTickInterval = 1 * time.Millisecond
+	sc.RaftTickInterval = 10 * time.Millisecond
 	sc.RaftHeartbeatIntervalTicks = 2
 	sc.RaftElectionTimeoutTicks = 10
 	sc.TestingKnobs.DisableScanner = true

